src/interop.rs

```diff
@@ -415,11 +415,12 @@ impl TryFrom<&tex::Entry> for Entry {
         }
 
         if let Some(eprint) = map_res(entry.eprint())? {
-            if map_res(entry.eprint_type().map(|c| c.format_verbatim().to_lowercase()))?
-                .as_deref()
-                == Some("arxiv")
-            {
+            let eprint_type =
+                map_res(entry.eprint_type().map(|c| c.format_verbatim().to_lowercase()))?;
+            if eprint_type == Some("arxiv".to_string()) {
                 item.set_arxiv(eprint);
+            } else if eprint_type == Some("pubmed".to_string()) {
+                item.set_pmid(eprint.replace("http://www.ncbi.nlm.nih.gov/pubmed/", ""));
```

Comment:
We could remove the `to_string` calls like so:

```suggestion
            let eprint_type =
                map_res(entry.eprint_type().map(|c| c.format_verbatim().to_lowercase()))?
                .as_deref();
            if eprint_type == Some("arxiv") {
                item.set_arxiv(eprint);
            } else if eprint_type == Some("pubmed") {
                item.set_pmid(eprint.replace("http://www.ncbi.nlm.nih.gov/pubmed/", ""));
```